### PR TITLE
chore(tile-info): update design of normals section

### PR DESCRIPTION
**Pull request by kirillovzah**

### CODE REVIEW

File: src/components/common.tsx [hunk 1]
```diff
@@ -266,3 +266,11 @@ export const MenuLink = styled.a`
...
   text-decoration: none;
 `;
+
+export const TileInfoSectionWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 24px 0;
+`;
```
Comment (dryabinin-actionengine) [inline src/components/common.tsx:276]: Please add empty line

**Review by dryabinin-actionengine — APPROVED** [review]

File: src/components/tile-details-panel/normals-input.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
```
Comment (belom88) [inline src/components/tile-details-panel/normals-input.tsx:1]: file name should be equal component's name: `normals-input-item.tsx`

Comment (belom88) [reply src/components/tile-details-panel/normals-input.tsx:1]: or rename the component

File: src/components/tile-details-panel/normals.tsx [hunk 3]
```diff
@@ -26,12 +29,42 @@ const NormalsContainer = styled.div`
...
+const NORMALS_PERCENTAGE_TITLE = "Percent of triangles with normals, %";
+const NORMALS_LENGTH_TITLE = "Normals length, m";
+
+type NormalsProps = {
+  tile: Tile3D;
+  trianglesPercentage: number;
+  normalsLength: number;
+  handleShowNormals: (tile: Tile3D) => void;
+  handleChangeTrianglesPercentage: (tile: Tile3D, percentage: number) => void;
+  handleChangeNormalsLength: (tile: Tile3D, length: number) => void;
```
Comment (belom88) [inline src/components/tile-details-panel/normals.tsx:41]: Callback name should be `onMakeSomething`. Handler might be `hanleMakeSomething` of `onMakeSomethingHandler`

**Review by belom88 — CHANGES_REQUESTED** [review]

**Review by belom88 — APPROVED** [review]

**Pull request merged**